engine/src/main/java/org/terasology/rendering/openvrprovider/ControllerListener.java

```diff
@@ -5,16 +5,16 @@
 
 /* Interface intended to be front-facing to user for controller interaction. */
 public interface ControllerListener {
-    public final int LEFT_CONTROLLER = 0;
-    public final int RIGHT_CONTROLLER = 1;
-    public static int k_EAxis_Trigger = 1;
-    public static int k_EAxis_TouchPad = 0;
-    public static long k_buttonTouchpad = (1L << JOpenVRLibrary.EVRButtonId.EVRButtonId_k_EButton_SteamVR_Touchpad);
-    public static long k_buttonTrigger = (1L << JOpenVRLibrary.EVRButtonId.EVRButtonId_k_EButton_SteamVR_Trigger);
-    public static long k_buttonAppMenu = (1L << JOpenVRLibrary.EVRButtonId.EVRButtonId_k_EButton_ApplicationMenu);
-    public static long k_buttonGrip = (1L << JOpenVRLibrary.EVRButtonId.EVRButtonId_k_EButton_Grip);
-    public static float triggerThreshold = .25f;
+    int LEFT_CONTROLLER = 0;
+    int RIGHT_CONTROLLER = 1;
+    int EAXIS_TRIGGER = 1;
```

Comment:
Any chance these names are made to match naming in another place, like the VR provider itself?

@indianajohn - ping! :-)